### DISCUSSION TITLE
How to install on Windows?

### DISCUSSION THREAD

**Discussion by dgmlr** [philippe44/AirConnect#508] [Q&A] [answered]
I'm stuck on this part of the instructions: "You will also need to grab the 2 dlls files"

What are the names of these files and where am I supposed to grab them from?

**Comment by philippe44** [comment]
They should be in the zip file

**Reply by dgmlr** [reply]
HI Phillippe, thanks for getting back to me.

That explains why I am stuck. There are no DLL files in the **AirConnect-1.5.4.zip** file that I downloaded.

In fact there are no **airupnp-win*** files that I was expecting from instructions. There is an **airupnp.exe** file; I'm guessing this is supposed to be the Windows executable?

**Reply by philippe44** [reply, marked as answer]
I've added them back

**Comment by dgmlr** [comment]
Thanks, I've got the DLL files and I'm now up and running!

**Comment by Master-Tea** [comment]
Double-click to run aircast.exe screen shows Start:735 Starting pico HTTP server on port xxxx and there is no following, how should I solve it
